src/utils/template_override_middleware.py

```diff
@@ -49,24 +49,24 @@ def get_theme_dirs(self):
                 try:
                     theme_setting = self.query_theme_dirs(_local.request.journal)
                 except BaseException:
-                    theme_setting = 'default'
+                    theme_setting = 'clean'
```

Comment:
Should be OLH, also replace BaseException above with Exception

Reply:
This is the default for journals only, I've changed the default for presses and added a migration for anyone who still has default set.